conductr-cloudformation.yaml

```diff
@@ -207,6 +274,10 @@ Resources:
         Timeout: 5
         UnhealthyThreshold: 2
         HealthyThreshold: 3
+      Instances:
```

Comment:
I presume this will fail as the nodes need at least 30 sec x 3 to become healthy.